### ISSUE TITLE
version 1.0.29 doesn't compile with typescript

### ISSUE THREAD

**Issue by AxelTerizaki** [http-party/node-portfinder#138]
Probably a syntax error somewhere that throws off the parser. Here's the error : 

```
node_modules/portfinder/lib/portfinder.d.ts:39:1 - error TS1128: Declaration or statement expected.

39 export setBasePort(port: number);
   ~~~~~~

node_modules/portfinder/lib/portfinder.d.ts:39:24 - error TS1005: ',' expected.

39 export setBasePort(port: number);
                          ~

node_modules/portfinder/lib/portfinder.d.ts:49:1 - error TS1128: Declaration or statement expected.

49 export setHighestPort(port: number);
   ~~~~~~

node_modules/portfinder/lib/portfinder.d.ts:49:27 - error TS1005: ',' expected.

49 export setHighestPort(port: number);
                             ~
```

**Comment by lelikg** [comment]
+1 having the same issue 
`node_modules/portfinder/lib/portfinder.d.ts(39,1): error TS1128: Declaration or statement expected.
node_modules/portfinder/lib/portfinder.d.ts(39,24): error TS1005: ',' expected.
node_modules/portfinder/lib/portfinder.d.ts(49,1): error TS1128: Declaration or statement expected.
node_modules/portfinder/lib/portfinder.d.ts(49,27): error TS1005: ',' expected.
npm ERR! code 2`

**Comment by jonluca** [comment]
Fixed in https://github.com/http-party/node-portfinder/pull/139

**Comment by eriktrom** [comment]
Taking a look, thanks, sorry was out last week

**Comment by eriktrom** [comment]
fixed by #141 

----

release scheduled for this morning
